server/fishtest/views.py

```diff
@@ -328,23 +328,35 @@ def sprt_calc(request):
 @view_config(route_name="actions", renderer="actions.mak")
 def actions(request):
     search_action = request.params.get("action", "")
-    search_user = request.params.get("user", "")
-    search_before = request.params.get("before", None)
-    try:
-        search_before = datetime.datetime.utcfromtimestamp(float(search_before))
-    except:
-        search_before = datetime.datetime.utcnow()
-    count = request.params.get("count", "100")
+    username = request.params.get("user", "")
+    timestamp = request.params.get("timestamp", None)
+
     try:
-        count = int(count)
+        timestamp = datetime.datetime.utcfromtimestamp(float(timestamp))
```

Comment:
DEV updated.
Nearly all our codebase is using the LBYL approach, so better to use the simple: 
```python
    if timestamp is not None:
        timestamp = datetime.datetime.utcfromtimestamp(float(timestamp))
```
In any case for the EAFP approach it's better to not use a bare exception:
```python
    try:
        timestamp = datetime.datetime.utcfromtimestamp(float(timestamp))
    except TypeError:
        pass
```
In some cases python requires EAFP, see:
https://realpython.com/python-lbyl-vs-eafp/